### ISSUE TITLE
Sometimes 'transport.LogStore' fails from 'too many open files' error.

### ISSUE THREAD

**Issue by evanlinjin** [skycoin/skywire-testnet#339]
```
[2019-05-09T11:22:39+12:00] WARN [trmanager]: Failed to record log entry: open: open localA/transport_logs/db2db6ba-96b7-5075-9e04-bc90fed323fb.log: too many open files
```

**Comment by evanlinjin** [comment]
This is not fixed. We need to have `LogStore` contain and maintain a map of open files. This way we won't have to constantly open and close files.

**Comment by jdknives** [comment]
Closed by #341
